tables/adverse-events/aet04.qmd

```diff
@@ -142,6 +147,7 @@ result <- lyt %>%
     decreasing = TRUE
   )
 
+result_v1 <- result
```

Comment:
this is an user-facing change that would be shown in the book - we don't want that
how about:
- remove that line while still keeping `result` object
- right below this code chunk open a new one with `echo=FALSE, message=FALSE` that saves the object to the desired path

This has also an advantage that we don't spread the logic throughout the script - e.g. if one add/remove a new variant (i.e. code chunk) then it can be completed in one part of the script.

Reply:
ahh ok I got what you did - you save _all_ variants in a _single_ file. I guess it would be fine to save it in multiple files using some naming convention

Reply:
I think it is fine too, it is just to avoid a bit of clatter, i.e. having like 3-6 files per table named like `aet04_variant1.Rdata` etc... Final system is the same. I also tried to hide the assignment of result_v1 and save it every time, but it is not in append and would require more computing time than this. We can still assign it in a dedicated hidden block after each result.